src/redux/hooks/hooks.ts

```diff
@@ -0,0 +1,6 @@
+import { useDispatch, useSelector } from "react-redux";
+import type { TypedUseSelectorHook } from "react-redux";
+import { AppDispatch, RootState } from "../store";
+
+export const useAppDispatch: () => AppDispatch = useDispatch;
```

Comment:
I know why you put this in _hooks/hoots.ts_ , but it can do in the _store.ts_ file.